radiomics/firstorder.py

```diff
@@ -283,10 +284,11 @@ def getUniformityFeatureValue(self):
     where a greater uniformity implies a greater heterogeneity or a
     greater range of discrete intensity values.
     """
+    eps = numpy.spacing(1)
 
     bins = imageoperations.getHistogram(self.binWidth, self.targetVoxelArray)[0]
     try:
-      bins = bins / (float(bins.sum()))
+      bins = bins / (float(bins.sum() + eps))
```

Comment:
I completely agree it should be handled uniformly, and also there should be some place in the documentation where the behavior is described. I personally prefer explicit NaN. I agree let's discuss at the next call.